src/SFA.DAS.PSRService.Web/Services/IReportService.cs

```diff
@@ -2,15 +2,17 @@
 using System.Collections.Generic;
 using SFA.DAS.PSRService.Domain.Entities;
 using SFA.DAS.PSRService.Domain.Enums;
-using SFA.DAS.PSRService.Web.Models;
-using SFA.DAS.PSRService.Web.ViewModels;
 
 namespace SFA.DAS.PSRService.Web.Services
 {
     public interface IReportService
     {
         void CreateReport(string employerId);
-        Report GetReport(string period,string employerId);
+
```

Comment:
Should this be rolled back or refactored?

Reply:
A start was made on replacing the 'stringly typed' GetReport method with one using the, newly introduced, Period object. However those changes were backed out, so we are left with an unused method. The work should be done to replace the stringly typed method, but it makes sense to remove it for now.

Removed.